hydra-node/src/Hydra/Ledger/Cardano/Configuration.hs

```diff
@@ -7,17 +8,17 @@ module Hydra.Ledger.Cardano.Configuration (
 import Hydra.Cardano.Api
 import Hydra.Prelude
 
+import Cardano.Ledger.BaseTypes (Globals (..), boundRational, mkActiveSlotCoeff)
 import qualified Cardano.Ledger.BaseTypes as Ledger
+import Cardano.Ledger.Shelley.API (computeRandomnessStabilisationWindow, computeStabilityWindow)
 import qualified Cardano.Ledger.Shelley.Genesis as Ledger
 import qualified Cardano.Ledger.Shelley.LedgerState as Ledger
 import qualified Cardano.Ledger.Shelley.Rules.Ledger as Ledger
-import Cardano.Slotting.EpochInfo (hoistEpochInfo)
+import Cardano.Slotting.EpochInfo (fixedEpochInfo)
 import Cardano.Slotting.Time (mkSlotLength)
-import Control.Arrow (left)
-import Control.Monad.Trans.Except (runExcept)
 import qualified Data.Aeson as Json
 import qualified Data.Aeson.Types as Json
-import qualified Ouroboros.Consensus.HardFork.History as Consensus
+import Hydra.Cardano.Api.Prelude (toShelleyNetwork)
```

Comment:
Don't use the Hydra.Cardano.Api.Prelude outside of hydra-cardano-api. It should be available on Hydra.Cardano.Api